tests/test_models.py

```diff
@@ -38,6 +41,15 @@ class TestSimplePicture:
         width=800,
     )
 
+    def test_hash(self):
+        assert hash(self.picture_with_ratio) != hash(self.picture_without_ratio)
+        assert hash(self.picture_with_ratio) == hash(self.picture_with_ratio)
+
+    def test_eq(self):
+        assert self.picture_with_ratio != self.picture_without_ratio
+        assert self.picture_with_ratio == self.picture_with_ratio
```

Comment:
Last asserts look weird tbh. 1 == 1 is not adding much value (except for looking consistent).

Reply:
I mean I am testing the equality function. So self == self is not a bad edge case to check.